scripts/zones/Apollyon/mobs/Fossil_Quadav.lua

```diff
@@ -0,0 +1,18 @@
+-----------------------------------
+-- Area: Apollyon CS
+--  MOB: Fossil_Quadav
+-----------------------------------
+package.loaded["scripts/zones/Apollyon/TextIDs"] = nil;
+-----------------------------------
+require("scripts/zones/Apollyon/TextIDs");
+require("scripts/globals/limbus");
+mixins = {require("scripts/mixins/job_special")};
+-----------------------------------
+
+function onMobFight(mob,target)
+end;
+
+function onMobDeath(mob, player, isKiller)
+    local helpers = GetServerVariable("deadQuadHelpers");
+    SetServerVariable("deadQuadHelpers",helpers +1);
```

Comment:
onMobDeath runs once for every party member. I don't think that's what you want to happen.